plugins/module_utils/network/junos/facts/bgp_global/bgp_global.py

```diff
@@ -66,6 +76,14 @@ def get_connection(self, connection, config_filter):
         """
         return connection.get_configuration(filter=config_filter)
 
+    def validate_config(self, spec, data, redact=False):
```

Comment:
This seems to be repeated for every module. We can move this to utils and pass the module object to it.